### ISSUE TITLE
unleash-server 6.4.0 Docker image is broken

### ISSUE THREAD

**Issue by danilofuchs** [Unleash/unleash#8601]
### Describe the bug

Cannot run unleash 6.4.0 in Docker container

### Steps to reproduce the bug

1. Follow the steps from the Docker guide
![image](https://github.com/user-attachments/assets/283cbf45-d8fc-4053-a56a-fd881f55c9a6)
2. Access localhost:4242

![image](https://github.com/user-attachments/assets/6a5d8405-6c52-4fb0-9076-4aef2f01a145)

### Logs, error output, etc.

```bash
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|   timestamp   |                                                                           message                                                                           |
|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 1730316552373 | [91m[2024-10-30T19:29:12.370] [ERROR] /debug-error-handler.ts - [39mServer failed executing request: {} TypeError: pathToRegexp.compile is not a function |
| 1730316552373 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:58:29                                                                                  |
| 1730316552373 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:71:3)                                                                    |
| 1730316552373 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:36                                                                                  |
| 1730316552373 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316552373 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:21)                                                                   |
| 1730316552373 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316552373 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316552373 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316552373 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316552373 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316552373 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316552373 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:17:5                                                                                   |
| 1730316552373 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316552373 |     at generateDocument (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:16:26)                                                               |
| 1730316552373 |     at OpenApiMiddleware (/unleash/node_modules/@wesleytodd/openapi/index.js:42:29)                                                                         |
| 1730316552373 |     at Layer.handle [as handle_request] (/unleash/node_modules/express/lib/router/layer.js:95:5)                                                            |
| 1730316552396 | [91m[2024-10-30T19:29:12.395] [ERROR] /debug-error-handler.ts - [39mServer failed executing request: {} TypeError: pathToRegexp.compile is not a function |
| 1730316552396 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:58:29                                                                                  |
| 1730316552396 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:71:3)                                                                    |
| 1730316552396 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:36                                                                                  |
| 1730316552396 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316552396 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:21)                                                                   |
| 1730316552396 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316552396 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316552396 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316552396 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316552396 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316552396 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316552396 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:17:5                                                                                   |
| 1730316552396 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316552396 |     at generateDocument (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:16:26)                                                               |
| 1730316552396 |     at OpenApiMiddleware (/unleash/node_modules/@wesleytodd/openapi/index.js:42:29)                                                                         |
| 1730316552396 |     at Layer.handle [as handle_request] (/unleash/node_modules/express/lib/router/layer.js:95:5)                                                            |
| 1730316612334 | [91m[2024-10-30T19:30:12.334] [ERROR] /debug-error-handler.ts - [39mServer failed executing request: {} TypeError: pathToRegexp.compile is not a function |
| 1730316612334 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:58:29                                                                                  |
| 1730316612334 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:71:3)                                                                    |
| 1730316612334 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:36                                                                                  |
| 1730316612334 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316612334 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:21)                                                                   |
| 1730316612334 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316612334 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316612334 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316612334 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316612334 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316612334 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316612334 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:17:5                                                                                   |
| 1730316612334 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316612334 |     at generateDocument (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:16:26)                                                               |
| 1730316612334 |     at OpenApiMiddleware (/unleash/node_modules/@wesleytodd/openapi/index.js:42:29)                                                                         |
| 1730316612334 |     at Layer.handle [as handle_request] (/unleash/node_modules/express/lib/router/layer.js:95:5)                                                            |
| 1730316612342 | [91m[2024-10-30T19:30:12.341] [ERROR] /debug-error-handler.ts - [39mServer failed executing request: {} TypeError: pathToRegexp.compile is not a function |
| 1730316612342 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:58:29                                                                                  |
| 1730316612342 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:71:3)                                                                    |
| 1730316612342 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:36                                                                                  |
| 1730316612342 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316612342 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:21)                                                                   |
| 1730316612342 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316612342 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316612342 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316612342 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316612342 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316612342 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316612342 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:17:5                                                                                   |
| 1730316612342 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316612342 |     at generateDocument (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:16:26)                                                               |
| 1730316612342 |     at OpenApiMiddleware (/unleash/node_modules/@wesleytodd/openapi/index.js:42:29)                                                                         |
| 1730316612342 |     at Layer.handle [as handle_request] (/unleash/node_modules/express/lib/router/layer.js:95:5)                                                            |
| 1730316672335 | [91m[2024-10-30T19:31:12.334] [ERROR] /debug-error-handler.ts - [39mServer failed executing request: {} TypeError: pathToRegexp.compile is not a function |
| 1730316672335 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:58:29                                                                                  |
| 1730316672335 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:71:3)                                                                    |
| 1730316672335 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:36                                                                                  |
| 1730316672335 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316672335 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:21)                                                                   |
| 1730316672335 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316672335 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316672335 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316672335 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316672335 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316672335 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316672335 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:17:5                                                                                   |
| 1730316672335 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316672335 |     at generateDocument (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:16:26)                                                               |
| 1730316672335 |     at OpenApiMiddleware (/unleash/node_modules/@wesleytodd/openapi/index.js:42:29)                                                                         |
| 1730316672335 |     at Layer.handle [as handle_request] (/unleash/node_modules/express/lib/router/layer.js:95:5)                                                            |
| 1730316672345 | [91m[2024-10-30T19:31:12.343] [ERROR] /debug-error-handler.ts - [39mServer failed executing request: {} TypeError: pathToRegexp.compile is not a function |
| 1730316672345 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:58:29                                                                                  |
| 1730316672345 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:71:3)                                                                    |
| 1730316672345 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:36                                                                                  |
| 1730316672345 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316672345 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:21)                                                                   |
| 1730316672345 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316672345 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316672345 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316672345 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316672345 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316672345 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316672345 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:17:5                                                                                   |
| 1730316672345 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316672345 |     at generateDocument (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:16:26)                                                               |
| 1730316672345 |     at OpenApiMiddleware (/unleash/node_modules/@wesleytodd/openapi/index.js:42:29)                                                                         |
| 1730316672345 |     at Layer.handle [as handle_request] (/unleash/node_modules/express/lib/router/layer.js:95:5)                                                            |
| 1730316732336 | [91m[2024-10-30T19:32:12.336] [ERROR] /debug-error-handler.ts - [39mServer failed executing request: {} TypeError: pathToRegexp.compile is not a function |
| 1730316732336 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:58:29                                                                                  |
| 1730316732336 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:71:3)                                                                    |
| 1730316732336 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:36                                                                                  |
| 1730316732336 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316732336 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:21)                                                                   |
| 1730316732336 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316732336 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316732336 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316732336 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316732336 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316732336 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316732336 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:17:5                                                                                   |
| 1730316732336 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316732336 |     at generateDocument (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:16:26)                                                               |
| 1730316732336 |     at OpenApiMiddleware (/unleash/node_modules/@wesleytodd/openapi/index.js:42:29)                                                                         |
| 1730316732336 |     at Layer.handle [as handle_request] (/unleash/node_modules/express/lib/router/layer.js:95:5)                                                            |
| 1730316732347 | [91m[2024-10-30T19:32:12.347] [ERROR] /debug-error-handler.ts - [39mServer failed executing request: {} TypeError: pathToRegexp.compile is not a function |
| 1730316732347 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:58:29                                                                                  |
| 1730316732347 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:71:3)                                                                    |
| 1730316732347 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:36                                                                                  |
| 1730316732347 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316732347 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:93:21)                                                                   |
| 1730316732347 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316732347 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316732347 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316732347 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:75:7                                                                                   |
| 1730316732347 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316732347 |     at iterateStack (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:73:24)                                                                   |
| 1730316732347 |     at /unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:17:5                                                                                   |
| 1730316732347 |     at Array.forEach (<anonymous>)                                                                                                                          |
| 1730316732347 |     at generateDocument (/unleash/node_modules/@wesleytodd/openapi/lib/generate-doc.js:16:26)                                                               |
| 1730316732347 |     at OpenApiMiddleware (/unleash/node_modules/@wesleytodd/openapi/index.js:42:29)                                                                         |
| 1730316732347 |     at Layer.handle [as handle_request] (/unleash/node_modules/express/lib/router/layer.js:95:5)                                                            |
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

### Unleash version

6.4.0

### Subscription type

None

### Hosting type

Self-hosted

**Comment by stevenchu31** [comment]
is this the fix? https://github.com/Unleash/unleash/pull/8597

**Comment by gastonfournier** [comment]
Hi @danilofuchs @stevenchu31 yes, that's the fix, I just applied it to the branch and the release 6.4.1 is about to be available

**Comment by gastonfournier** [comment]
New image got published under latest, 6 and 6.4 tags (besides 6.4.1)
